release_notes/Service_Fabric_ReleaseNotes_91CU4.md

```diff
@@ -0,0 +1,62 @@
+|Java SDK  |Java for Linux SDK  | 1.0.6 |
+|Service Fabric PowerShell and CLI | AzureRM PowerShell Module  <br> SFCTL |  0.3.15  <br> 11.0.1 |
+
+
+## Service Fabric Feature and Bug Fixes
+| Versions | IssueType | Description | Resolution | 
+|-|-|-|-|
+| **Windows - 9.1.1799.9590<br>Ubuntu 18 - 9.1.1592.1<br>Ubuntu 20 - 9.1.1592.1** | **Feature** | Key Value Store (KVS) | **Brief Description**:  Auto-compaction now emits health events at the start and completion, visible in the Service Fabric Explorer, enhancing its monitoring capabilities. These events complement trace data, providing effective visibility into active auto-compaction progress.
+| **Windows - 9.1.1799.9590<br>Ubuntu 18 - 9.1.1592.1<br>Ubuntu 20 - 9.1.1592.1** | **Feature** | Key Value Store (KVS) | **Brief Description**: KVS backed partitions, including Actors and System Services, experience quorum loss due to StoreLVIDLimitHit error code (0x80071d4d), indicating the depletion of values for a monotonically increasing ID field. This results in unavailability of the partition until a manual offline maintenance task is executed to resolve the issue.<br>**Solution**: To address the error, KVS replicas now restart automatically and initiate the maintenance task (Extensible Storage Engine database compaction) during the next open. While there may be temporary availability loss in partitions depending on the task duration, no manual intervention is required to mitigate the issue.
+| **Windows - 9.1.1799.9590<br>Ubuntu 18 - 9.1.1592.1<br>Ubuntu 20 - 9.1.1592.1** | **Feature** | Key Value Store (KVS) | **Brief Description**: The "IntrinsicValueThresholdInBytes" setting controls the size of long values stored in a record. If a value exceeds this size, it is stored as an Long Value ID (LVID) within the record. Presently, the default value is 0 which sets the size to 1024 bytes. However, in the future, the value will be changed to 5120 bytes to enable storage of long values up to approximately 5KB within a record. This change is expected to minimize LVID creation and mitigate the risk of LVID exhaustion. <br>**Solution**: The default value of the IntrinsicValueThresholdInBytes is set to 5120 bytes.
+| **Windows - 9.1.1799.9590<br>Ubuntu 18 - 9.1.1592.1<br>Ubuntu 20 - 9.1.1592.1** | **Feature** | Application Model | **Brief Description**: Service Fabric has a restriction that allows only one application type with the same name to be provisioned or unprovisioned at any given time. However, if an application type contains illegal path characters in its name or version, it becomes stuck in an unprovisioning loop. This situation not only affects the problematic application type but also causes other application types with the same name to fail during provisioning. <br>**Solution**:  Improvements were made in Service Fabric to address the issue with invalid characters. 
```

Comment:
Do we have docs showing what valid characters are? May be good to link to them from here.